clusters/l3-sqnc/capacity/nginx/values.yaml

```diff
@@ -0,0 +1,34 @@
+watchIngressWithoutClass: false
+scope:
+  enabled: true
+ingressClassResource:
+  name: nginx-capacity
+  default: false
+config:
+    global-auth-url: "http://oathkeeper-api.ory.svc.cluster.local/decisions$request_uri"
+service:
+  type: LoadBalancer
+  annotations: 
+    service.beta.kubernetes.io/azure-load-balancer-health-probe-request-path: /healthz
+    external-dns.alpha.kubernetes.io/hostname: capacity.dsch-l3.com
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    cert-manager.io/common-name: capacity.dsch-l3.com
```

Comment:
Remove as cert-manager is not yet installed.

Reply:
this as well. other that required removal will mark as resolved with a comment.